packages/fhir-keycloak-user-management/src/components/UserList/TableActions/index.tsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import * as React from 'react';
+import { Popconfirm, Divider, Dropdown, Menu, Button } from 'antd';
+import { MoreOutlined } from '@ant-design/icons';
+import { deleteUser } from './utils';
+import { Link } from 'react-router-dom';
+import { removeKeycloakUsers, KeycloakUser } from '../../../ducks/user';
+import { URL_USER_CREDENTIALS, URL_USER_EDIT } from '../../../constants';
+import { Dictionary } from '@onaio/utils';
+import lang from '../../../lang';
+
+export interface Props {
+  removeKeycloakUsersCreator: typeof removeKeycloakUsers;
+  keycloakBaseURL: string;
+  fhirBaseURL: string;
+  record: KeycloakUser;
+  isLoadingCallback: (loading: boolean) => void;
+  extraData: Dictionary;
+}
+
+/**
+ * Component TableActions
+ *
+ * @param {React.PropsTypes} props - component props
+ * @returns {Element} - actions
+ */
+const TableActions = (props: Props): JSX.Element => {
```

Comment:
Some of these components are very similar to the ones used in the other user management package, how can we dry this out